plugins/toolbox/src/components/Encoders/JwtDecoder.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { useEffect } from 'react';
+import jwtDecode from 'jwt-decode';
+import { DefaultEditor } from '../DefaultEditor/DefaultEditor';
+
+export const JwtDecoder = () => {
+  const [input, setInput] = React.useState('');
+  const [output, setOutput] = React.useState('');
+
+  const exampleJwt =
+    'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIiwiaWF0IjoxNTE2MjM5MDIyLCJleHAiOjE1MTYyMzkwMjJ9.4Adcj3UFYzPUVaVF43FmMab6RlaQD8A9V8wFzzht-KQ';
+
+  useEffect(() => {
+    if (input) {
+      const jwtPayload = jwtDecode(input) as { iat?: number; exp?: number };
```

Comment:
Maybe this needs error handling in case of invalid jwt token? Otherwise looks promising!

Reply:
Implemented simple error handling.